nyc.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  reporter: ["text", "lcov"],
+  include: ["out/src/**"],
+  exclude: ["out/src/logger/**"],
+  // https://reflectoring.io/100-percent-test-coverage/
+  branches: 85,
+  lines: 90,
+  functions: 85,
+  statements: 90,
+  "check-coverage": true,
+  excludeAfterRemap: false,
```

Comment:
Please remove this, it's not necessary here. It's done specifically in hana explorer for vscode tests in a vscode environment.

Reply:
Without it the coverage is not presented

Reply:
ok